app/models/geolocation.rb

```diff
@@ -0,0 +1,91 @@
+require Rails.root.join("lib/ISO3166/country")
```

Comment:
I would recommend putting this in something similar to `config/initializer/countries.rb`. This way it's guarantee this gem is globally monkey-patched on boot and not only after this file is loaded. 

You can check `config/initializers/liquid.rb` for inspiration.